### DISCUSSION TITLE
When Do I use Prefix and when Do I use IP range

### DISCUSSION THREAD

**Discussion by Riprock** [netbox-community/netbox#14404] [Q&A]
Im just curious how do I organize IP addresses when im seeing in the documentation For IP ranges: "IP also ranges share the same functional roles as prefixes and VLANs, although the assignment of a role is optional". I'm just confused on the distinction between the two classifications

**Comment by jeremystretch** [comment]
A prefix represents a subnet defined by a mask, e.g. 192.168.0.0/24. A range represents a set of individual IP addresses inclusive of a starting and ending address, e.g. 192.168.0.100/24 through 192.168.0.150/24.

Prefixes typically represent routed allocations, whereas ranges document arbitrary designations such as DHCP ranges. Hope that helps!

**Comment by gdprdatasubect** [comment]
A range can be used for further dividing your net without using subnet masks, e.g. you can define x.y.z.0-10 as printservers in your firewall, while x.y.z.11-20 are used for printers themselfs etc.
